components/server/src/ome/services/sharing/ShareBean.java

```diff
@@ -752,6 +757,35 @@ public void invalidateConnection(long shareId, Experimenter exp) {
         throw new UnsupportedOperationException();
     }
 
+    @RolesAllowed("user")
+    @Transactional(readOnly = false)
+    public void notifyMembersOfShare(long shareId, String subject, String message,
+            boolean html, List<Experimenter> exps) {
+
+        String sender = mailUtil.getSender();
+        if (sender.length() < 1) {
```

Comment:
similarly is an `isEmpty()` on strings now too, though if it might have whitespace and that's bad you can also use `StringUtils.isBlank`